DigitalLearningSolutions.Web/Views/LearningPortal/SelfAssessments/ManageSupervisors.cshtml

```diff
@@ -48,31 +45,17 @@
               @supervisor.SupervisorName, @supervisor.RoleName
 
             </td>
-            <td class="nhsuk-table__cell">
-              <span class="nhsuk-table-responsive__heading">Status </span>
-              @if (supervisor.Confirmed == null)
-              {
-                <span class='nhsuk-tag nhsuk-tag--grey'>Unconfirmed</span>
-              }
-              else
-              {
-                <span class='nhsuk-tag'>Confirmed</span>
-              }
-            </td>
             <td class="nhsuk-table__cell">
               <span class="nhsuk-table-responsive__heading">Actions </span>
               @if (supervisor.AddedByDelegate)
               {
-                <a asp-action="RemoveSupervisor" asp-route-selfAssessmentId="@Model.SelfAssessment.Id" asp-route-supervisorDelegateId="@supervisor.ID">Remove</a>
+                <a asp-action="RemoveSupervisor" asp-route-selfAssessmentId="@Model.SelfAssessment.Id" asp-route-supervisorDelegateId="@supervisor.ID">Remove</a>
+                @Html.Raw("&nbsp;")
                 @if (supervisor.Confirmed == null && supervisor.NotificationSent.ToShortDateString() != DateTime.UtcNow.ToShortDateString())
```

Comment:
We shouldn't need the send reminder functionality either.

Reply:
Done. Added new commit for removing "send reminder" link in Manage Supervisors.